exercises/series/canonical-data.json

```diff
@@ -0,0 +1,73 @@
+{
+    "exercise": "series",
+    "version": "1.0.0",
+    "cases": [
+        {
+            "description": "slices of one",
+            "property": "slices",
+            "series": "01234",
```

Comment:
We are in the processing of ensuring that all input properties are gathered in a single `input` key (see #996). Could you put the `series` and `slice_length` properties in an `input` field (and also in the other cases)?